### PR TITLE
Review

### PR DESCRIPTION
**Pull request by anatolydyachenko**

### CODE REVIEW

File: src/main/java/app/Product.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    public Double getPrice() {
+        return price;
+    }
+
+    public void setPrice(Double price) {
+        this.price = price;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```
Comment (ikukarin) [inline src/main/java/app/Product.java:41]: don't understand

File: src/main/java/app/ProductStorage.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+            availableProductsListForJson.add(storeProduct);
+        }
+
+        return mapToJson(availableProductsMapForJson);
+    }
+
+    public void addProduct(Product product, Integer count) {
+        int existingProducts = 0;
+        Object[] productFound = availableProducts.stream().filter(row -> row[0] == product).findFirst().orElse(null);
```
Comment (ikukarin) [inline src/main/java/app/ProductStorage.java:36]: Too massive construction. I would suggest to breaking up

File: src/main/java/app/Product.java (hunk 1 above)
Comment (anatolydyachenko) [reply src/main/java/app/Product.java:41]: equals method was overridden from Object implementation

Comment (anatolydyachenko) [reply src/main/java/app/Product.java:41]: btw, method was generated by idea

**Pull request closed without merging**